site/en/docs/extensions/reference/action/index.md

```diff
@@ -0,0 +1,7 @@
+---
+api: action
+---
+
+!!!.aside.warning
+This documentation is in active development. We're planning to add an expanded introduction section soon to provide better context and how-to focus. This work is tracked in issue [#193](https://github.com/GoogleChrome/developer.chrome.com/issues/193).
```

Comment:
I don't really follow this phrasing: "provide better context and how-to focus". It seems like something is missing to set up or contextualize the note about "how-to fous". 

What do you think about replacing the current sentence with this: "We're planning to expand the introduction section to provide more context and guidance on how to use this API."

Reply:
Done. PTAL.